app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -77,7 +77,7 @@ fun WiFiIconText() {
         ),
             contentDescription = "WiFi Icon",
             modifier = Modifier
-                .size(30.dp)
+                .size(44.dp)
```

Comment:
Changing size is a bit resolution dependent, and a design consideration.  It doesn't necessarily decrease technical debt, unless there is an accessibility issue.